features/page_objects/back_office/new/registration_edit_completed_page.rb

```diff
@@ -0,0 +1,5 @@
+class RegistrationEditCompletedPage < SitePrism::Page
```

Comment:
You could avoid creating this file by using `@journey.standard_page` then `find_link("View registration").click`